app/views/posts/display.html.erb

```diff
@@ -0,0 +1 @@
+<h2>Here is a list of posts for a given user</h2>
```

Comment:
- [ ] Kindly rename this file to match the action renamed in the controller: `show.html.erb`